great_expectations/execution_engine/sqlalchemy_batch_data.py

```diff
@@ -239,6 +239,9 @@ def _create_temporary_table(
                 temp_table_name=temp_table_name
             )
         elif self.sql_engine_dialect.name.lower() == "awsathena":
+            logger.warning(
```

Comment:
@Shinnnyshinshin Can we put a "note" here (not actual code) that for `trino` logic such as this warning might be needed in the future, but we are not prepared to do it yet.  Thanks!